drake/solvers/MultipleTreeProblem.m

```diff
@@ -0,0 +1,463 @@
+classdef MultipleTreeProblem
+  
+  properties
+    robot
+    endEffectorId
+    xStart
+    xGoal
+    endEffectorPoint
+    goalConstraints
+    additionalConstraints
+    qNom
+    mergingThreshold
+    capabilityMap
+    graspingHand
```

Comment:
the capability map and grasping hand seem very humanoids specific, where the rest of the code need not be.  is there a way to force that separation?